src/controllers/confirmation.controller.ts

```diff
@@ -2,13 +2,17 @@ import { Session } from "ch-node-session-handler";
 import { NextFunction, Request, Response } from "express";
 import { SESSION_OBJECTION_ID } from "../constants";
 import { Templates } from "../model/template.paths";
-import { retrieveFromObjectionSession, retrieveUserEmailFromSession } from "../services/objection.session.service";
+import { deleteFromObjectionSession, retrieveFromObjectionSession, retrieveUserEmailFromSession } from "../services/objection.session.service";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   const session: Session | undefined  = req.session;
   if (session) {
     const email: string = retrieveUserEmailFromSession(session);
     const objectionId: string = retrieveFromObjectionSession(session, SESSION_OBJECTION_ID);
+
+    // Objection has been submitted and should no longer be referenced by this web session, so remove the objection id from the session
+    deleteFromObjectionSession(session, SESSION_OBJECTION_ID);
```

Comment:
This addresses the additional issue highlighted in Dale’s final comment on OBJ-233 - navigating around the web app by changing URLs and being able to change the Objection Mongo DB data after it’s been submitted.